utils.groovy

```diff
@@ -199,4 +199,29 @@ def get_push_trigger() {
     ]
 }
 
+def get_artifacts(pipecfg, stream, basearch) {
+    // Get artifacts from config.yaml
+    def artifacts
+    if  (pipecfg.streams."${stream}".artifacts?."${basearch}" != null) {
+        artifacts = pipecfg.streams."${stream}".artifacts."${basearch}"
+    }
+    else { // Merge default with additional artifacts
+        def default_artifacts =  pipecfg.default_artifacts."${basearch}"
+        def additional_atifacts = pipecfg.streams."${stream}".additional_artifacts?."${basearch}"
+        artifacts = default_artifacts + additional_atifacts
+    }
+    // For all architectures we need to build the metal/metal4k artifacts and the Live ISO. Since the
+    // ISO depends on the Metal/Metal4k images we'll make sure to put the Metal* ones in the first run
+    // and the Live ISO in the second run for parallel runs where the list is split
+    if (artifacts.contains('Metal') && artifacts.contains('Live')) {
```

Comment:
I think this can just be

```suggestion
    if (artifacts.contains('Live')) {
```

Reply:
True, since Live depends on Metal we can assume it exists?